### PR TITLE
docs: Add Skia to the issue templates

### PR DESCRIPTION
**Pull request by dr1rrb**
## Docs
Add Skia to the issue templates

## PR Checklist
- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Affected platform(s):
 - [ ] WebAssembly
 - [ ] WebAssembly renderers for Xamarin.Forms
 - [ ] macOS
+- [ ] Skia
```
Comment (davidjohnoliver) [inline .github/ISSUE_TEMPLATE/bug-report.md:39]: Should we have both `[ ] Skia (Linux)` and `[ ] Skia (WPF)`?

We know that they're pretty similar, but the person reporting doesn't necessarily know, and they may be behaving differently in some cases.

Comment (dr1rrb) [reply .github/ISSUE_TEMPLATE/bug-report.md:39]: Good point ... we might end with an amazingly high number of targets, but for now it makes sense. Will change that.

**Comment by MartinZikmund** [comment]
@dr1rrb Oh, I have this already in #3845 with some additional changes. Can you check?

**Comment by dr1rrb** [comment]
> 
> 
> @dr1rrb Oh, I have this already in #3845 with some additional changes. Can you check?

@MartinZikmund good 👍, as I'm not able to push my changes to your branch, I chose the easy way to replicate your changes here

**Comment by MartinZikmund** [comment]
Thanks, I will close the PR 👌

**Pull request merged**